command_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package updater
+
+import (
+	"os"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/keybase/go-logging"
+)
+
+var log = logging.Logger{Module: "test"}
+
+func TestEmptyRunCommand(t *testing.T) {
+	out, err := RunCommand("", nil, time.Second, log)
```

Comment:
Yes but if the tests currently run in 500ms, you can run them on every save. Adding 4 seconds makes that impossible. We should really try to avoid adding sleeps to tests if at all possible, especially for such workload-light tests.